pkg/pillar/types/downloadertypes.go

```diff
@@ -167,3 +168,71 @@ func AllowNonFreePort(gc ConfigItemValueMap, objType string) bool {
 		return false
 	}
 }
+
+// ResolveConfig key/index to this is the ImageID which is allocated by the controller.
+// It will resolve container image tag to sha256
+type ResolveConfig struct {
+	AppID            uuid.UUID
+	ImageID          uuid.UUID
```

Comment:
I thought we agreed to get rid of these.

Reply:
Resolved.